build.conf.arm

```diff
@@ -0,0 +1 @@
+COMPILED_KERNEL_URL=https://github.com/rancher/os-kernel/releases/download/HypriotOS-4.1.12-v7+/linux-4.1.12-hypriotos-v7.-arm.tar.gz
```

Comment:
@imikushin I think the standard Raspbian kernel is still missing some important Docker tuning settings. But feel free to ask. It's not too complicated to customise and compile your own kernel for the Raspberry Pi. We've just open sourced all of these bits in our repo https://github.com/hypriot/rpi-kernel and you could build your own kernel pretty easily as well, just modify the .config and build it. :smirk: